### DISCUSSION TITLE
Activate / Deactivate Profile

### DISCUSSION THREAD

**Discussion by PatrickGlatz** [pivpn/pivpn#1471] [Feature requests]
Hello pivpn Team,

thanks for the great work. Love to have pivpn with me.

A feature which I am missing is the possibility to activate and deactivate WireGuard profiles. The only way is always to create a profile, share it and then remove it. But what i need is to give temporarily but repeatedly access from time to time to my mother or my brother in law or a friend of mine, for example to grab some files. By always creating-sharing-removing this is very inconvenient. 

Is there a chance to have an option implemented to activate and deactivate a WireGuard profile? 

Thanks in advance 
Patrick

**Comment by orazioedoardo** [comment]
It’s already implemented via `pivpn on`  and `pivpn off`.

**Reply by PatrickGlatz** [reply]
Classic example for rtfm. Thanks a lot, working. Topic can be closed.
